EIPS/eip-1753.md

```diff
@@ -10,7 +10,7 @@ created: 2019-02-06
 
 ## Abstract
 
-This Ethereum Improvement Proposal (EIP) proposes an Ethereum standard for the issuance of licences, permits and grants (Licences). 
+This Ethereum Improvement Proposal (EIP) proposes an Ethereum standard for the issuance of licences, permits and grants (Licences) as a Non Transferable Token NTT. 
```

Comment:
Hi @bitcoinbrisbane , welcome back to EIP process. This EIP has been Stagnant for a while, if you intend to revive this EIP and resume the process to move forward, can you update the status back to Draft and then fix all linter errors?
```suggestion
This Ethereum Improvement Proposal (EIP) proposes an Ethereum standard for the issuance of licences, permits and grants (Licences) as a Non Transferable Token (NTT). 
```